gpkit/nomials.py

```diff
@@ -517,29 +507,37 @@ def __repr__(self):
     def _latex(self, unused=None):
         return self.left._latex() + self.oper_l + self.right._latex()
 
-    def __init__(self, p1, p2):
-        """Initialize a constraint of the form p2 >= p1.
+    def __init__(self, left, right, oper_le=True):
+        """Initialize a constraint of the form left <= right
+        (or left >= right, if oper_le is False).
 
         Arguments
         ---------
-        p1 (Signomial)
-        p2 (Signomial)
+        left: Signomial
+        right: Signomial
+        oper_le: bool
+            If true, form is left <= right; otherwise, left >= right.
+
+        Note: Constraints initialized via operator overloading always take
+              the form left <= right, e.g. (x >= y) becomes (y <= x).
 
         TODO: clarify how this __init__ handles Signomial constraints
-        TODO: change p1 and p2 to left and right instead of auto-selecting?
-        TODO: call super() from this __init__
+              (may want to create a SignomialConstraint class that does not
+               inherit from Posynomial and keeps left and right separate).
```

Comment:
Although that probably makes sense technically, I didn't intend it, and I think it would be a challenge. `SignomialConstraint` would not be able to inherit from `Posynomial`. We'd thus have a double-inheritance situation (`Constraint(Posynomial, SignomialConstraint)`), and I hear python doesn't play too well with multiple inheritance.